.github/workflows/tests.yml

```diff
@@ -37,14 +37,14 @@ jobs:
         with:
           path: ~/.cache/pip
           key: |
-            ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('setup.py', 'docs/requirements.txt') }}
+            ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('pyproject.toml') }}
           restore-keys: |
             ${{ runner.os }}-pip-${{ matrix.python-version }}-
             ${{ runner.os }}-pip-
```

Comment:
If you're touching this, you can probably drop this whole thing and use setup-python v2 and provide `cache: pip` to it. https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
Ah, yes! I saw that passing by on twitter :)